### DISCUSSION TITLE
Complete uninstall

### DISCUSSION THREAD

**Discussion by geir55** [elden1337/hass-peaq#360] [Q&A] [answered]
Hello.
Im not using this Integration anymore.

How do i do a **complete** uninstall of integration, without any complicatons to EV charger etc.?

Mvh
Geir A

**Comment by elden1337** [comment, marked as answer]
Hi. 
If you uninstall it from home assistant and reboot it should not be present any longer. 
I have not altered the charger itself, everything happened only locally within the application so uninstalling that should be enough.
